libraries/matrix/test/src/main/kotlin/io/element/android/libraries/matrix/test/room/FakeMatrixRoom.kt

```diff
@@ -84,126 +84,88 @@ class FakeMatrixRoom(
     override val activeMemberCount: Long = 234L,
     val notificationSettingsService: NotificationSettingsService = FakeNotificationSettingsService(),
     override val liveTimeline: Timeline = FakeTimeline(),
-    private var roomPermalinkResult: () -> Result<String> = { Result.success("room link") },
-    private var eventPermalinkResult: (EventId) -> Result<String> = { Result.success("event link") },
-    var sendCallNotificationIfNeededResult: () -> Result<Unit> = { Result.success(Unit) },
-    canRedactOwn: Boolean = false,
-    canRedactOther: Boolean = false,
+    private var roomPermalinkResult: () -> Result<String> = { lambdaError() },
```

Comment:
I tend to make these public so we don't have to add extra `given___` methods to modify them, but it's no big deal.